src/analysis/normed_space/star/spectrum.lean

```diff
@@ -23,13 +23,14 @@ open filter ennreal spectrum cstar_ring
 section unitary_spectrum
 
 variables
-{𝕜 : Type*} [normed_field 𝕜]
+{𝕜 : Type*} [nontrivially_normed_field 𝕜]
```

Comment:
Can you explain this change in the commit message? At a glance, this appears to be making the lemmas that follow less general.

Reply:
Thanks, fixed. It was because the `nontriviality E` was using a result from `analysis/normed_space/spectrum` and that had an unnecessary `nontrivially_normed_field` hypothesis.

Reply:
Just to clarify in case anyone is interested: I was originally going to go about this refactor the standard mathematical way, which is to show that the algebra is equivalent (bilipschitz and algebra) to its left-regular representation, for which the `norm_one_class` is satisfied and then pull the theorems back. I did the initial work to make this happen, but then decided it would be easier / better to go the more direct way, as I have done here. But then I forgot to remove the `nontrivially_normed_field` hypotheses that I needed for that equivalence.